services/order.py

```diff
@@ -0,0 +1,27 @@
+from django.db import transaction
+from db.models import Order, Ticket, User
+from django.db.models import QuerySet
+
+
+@transaction.atomic
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: str = None) -> Order:
+    user = User.objects.get(username=username)
```

Comment:
Consider adding error handling for the case where the `User` object with the given `username` does not exist. This will prevent the function from raising an unhandled exception.